### DISCUSSION TITLE
ReBAR on Maxwell 2 (GTX 970)?

### DISCUSSION THREAD

**Discussion by TheLynxium** [terminatorul/NvStrapsReBar#32] [Q&A]
I just finished doing the ReBAR UEFI BIOS mod, just to realize the GTX 970 I'm running doesn't support it. 
Would this tool work with a GTX 970 in some way, or is it just too old for it?

**Comment by terminatorul** [comment]
Someone else tried and it didn't work https://github.com/terminatorul/NvStrapsReBar/discussions/28

Maxwell 2 cards do not have the PCI Resizable BAR capability, but NvStrapsReBar can still make the card report a large BAR (even if fixed or non-resizable).

But then the NVIDIA driver for Windows refuzes to work the card, and the GPU appears in Device Manager with a problem code.

**Comment by D337z** [comment]
I have an odd thought here that I'm curious as to whether anyone has tried.  The cards that don't seem to work with the patch just happen to be the same ones that run in a mode requiring CSM to be enabled by default but also have a BIOS update for them to fix the issue.  So, my question is, has anyone tried using the Displayport 1.3/1.4 BIOS update for the GPUs to see if they'll allow the card to run in the correct mode to enable the patch?  If the system can't find the card using the correct ID system, it won't work.
